incubator/hnc/cmd/manager/main.go

```diff
@@ -199,3 +203,22 @@ func startControllers(mgr ctrl.Manager, certsCreated chan struct{}) {
 
 	setupLog.Info("All controllers started; setup complete")
 }
+
+// arrayArg is an arg that can be specified multiple times. It implements
+// https://golang.org/pkg/flag/#Value an is based on
+// https://stackoverflow.com/questions/28322997/how-to-get-a-list-of-values-into-a-flag-in-golang.
```

Comment:
Per "can be specified multiple times", does it mean people can both insert or delete annotation from the array? Seems there's no deletion behavior in the code?

s/an/and

Reply:
No, this is only specified during startup so there's no "delete" behaviour. It just means that you can have the flag repeated multiple times, instead of only having the flag once and making it comma-delimited.

Reply:
sg!